js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/AssetAutomaterializePolicyPage.tsx

```diff
@@ -24,92 +98,236 @@ export const AssetAutomaterializePolicyPage = () => {
-            {formattedDate}
-            {evaluation.materialized ? (
-              <Icon name="auto_materialize_policy" size={24} />
-            ) : (
-              <div
-                style={{borderRadius: '50%', height: '10px', width: '10px', color: Colors.Yellow50}}
-              />
-            )}
-          </CenterAlignedRow>
-        );
-      })}
+    <Box flex={{direction: 'column', grow: 1}} style={{overflowY: 'scroll'}}>
+      <Box
+        style={{flex: 1, minHeight: 0, overflowY: 'scroll'}}
+        flex={{grow: 1, direction: 'column'}}
+      >
+        {evaluations.map((evaluation) => {
+          const date = dayjs.unix(evaluation.timestamp);
+          const formattedDate = date.format('lll');
+          const isSelected = selectedEvaluation === evaluation;
+          return (
+            <EvaluationRow
+              flex={{justifyContent: 'space-between'}}
+              key={evaluation.evaluationId + '' + Math.random()}
+              padding={{horizontal: 24, vertical: 8}}
+              border={{side: 'bottom', width: 1, color: Colors.KeylineGray}}
+              onClick={() => {
+                onSelectEvaluation(evaluation);
+              }}
+              $selected={isSelected}
+            >
+              {formattedDate}
+              {evaluation.numRequested ? (
+                <Icon name="auto_materialize_policy" size={24} />
+              ) : evaluation.numSkipped ? (
+                <div
+                  style={{
+                    borderRadius: '50%',
+                    height: '10px',
+                    width: '10px',
+                    color: Colors.Yellow500,
+                  }}
+                />
+              ) : (
+                <div
+                  style={{
+                    borderRadius: '50%',
+                    height: '10px',
+                    width: '10px',
+                    color: Colors.Yellow50,
+                  }}
+                />
+              )}
+            </EvaluationRow>
+          );
+        })}
+      </Box>
+      {(queryResult.data?.autoMaterializeAssetEvaluations?.length || 0) > 0 ? (
+        <PaginationWrapper>
+          <CursorPaginationControls {...paginationProps} />
+        </PaginationWrapper>
+      ) : null}
     </Box>
   );
-};
+}
+
+const RightPanel = ({
+  assetKey,
+  setMaxMaterializationsPerMinute,
+}: {
+  assetKey: Omit<AssetKey, '__typename'>;
+  setMaxMaterializationsPerMinute: (max: number) => void;
+}) => {
+  const queryResult = useQuery<GetPolicyInfoQuery, GetPolicyInfoQueryVariables>(
+    GET_POLICY_INFO_QUERY,
+    {
+      variables: {
+        assetKey,
+      },
+    },
+  );
+  useQueryRefreshAtInterval(queryResult, FIFTEEN_SECONDS);
+  const {data, error} = queryResult;
+
+  React.useEffect(() => {
+    if (data?.assetNodeOrError.__typename === 'AssetNode') {
+      const max = data.assetNodeOrError.autoMaterializePolicy?.maxMaterializationsPerMinute;
+      if (max !== undefined && max !== null) {
```

Comment:
`if (typeof max === 'number') {`? Also, could `max` be zero, and if so, is that a problem?

Reply:
not a problem.